config/projects/G3g/RobotTest_config.json

```diff
@@ -0,0 +1,3 @@
+{
+	"Name": "G3G Configuration"

```

Comment:
Hi Nguyen,
G3g is Bosch internal.
Therefore the best is only to have a generic configuration for 0.7.0 version.
For 0.8.0 we look how to improve.
Thank you,
 Thomas

Reply:
Hi Thomas,
The internal project configs is removed.